pages/books.tsx

```diff
@@ -53,9 +57,20 @@ const formatDate = (date: string): string => {
 export const getServerSideProps = async () => {
   try {
     const url = getEnvUrl();
-    console.log(`${url}/api/books`);
-    const response = await axios.get(`${url}/api/books`);
-    const books = response.data;
+
+    const [responseBooks, responseRegions, responseLocations, responseGenres] =
+      await Promise.all([
+        axios.get(`${url}/api/books`),
+        axios.get(`${url}/api/regions`),
+        axios.get(`${url}/api/locations`),
+        axios.get(`${url}/api/genres`)
+      ]);
```

Comment:
This works for now, but I'll just be honest that this doesn't seem great in the long term. It may be better to have an API call that does all of this so only one request is made.